test/document-util.test.js

```diff
@@ -110,165 +110,176 @@ function findImposterElement(document) {
     return document.querySelector('div[style*="2147483646"]>*');
 }
 
-const test = createDomTest(path.join(dirname, 'data/html/document-util.html'));
+const documentUtilTestEnv = await setupDomTest(path.join(dirname, 'data/html/document-util.html'));
 
-describe('DocumentUtil', () => {
-    test('Text scanning functions', ({window}) => {
-        const {document} = window;
-        for (const testElement of /** @type {NodeListOf<HTMLElement>} */ (document.querySelectorAll('test-case[data-test-type=scan]'))) {
-            // Get test parameters
-            /** @type {import('test/document-util').DocumentUtilTestData} */
-            const {
-                elementFromPointSelector,
-                caretRangeFromPointSelector,
-                startNodeSelector,
-                startOffset,
-                endNodeSelector,
-                endOffset,
-                resultType,
-                sentenceScanExtent,
-                sentence,
-                hasImposter,
-                terminateAtNewlines
-            } = parseJson(/** @type {string} */ (testElement.dataset.testData));
+describe('Document utility tests', () => {
+    const {window, teardown} = documentUtilTestEnv;
+    afterAll(() => teardown(global));
 
-            const elementFromPointValue = querySelectorChildOrSelf(testElement, elementFromPointSelector);
-            const caretRangeFromPointValue = querySelectorChildOrSelf(testElement, caretRangeFromPointSelector);
-            const startNode = getChildTextNodeOrSelf(window, querySelectorChildOrSelf(testElement, startNodeSelector));
-            const endNode = getChildTextNodeOrSelf(window, querySelectorChildOrSelf(testElement, endNodeSelector));
+    describe('DocumentUtil', () => {
+        describe('Text scanning functions', () => {
+            let testIndex = 0;
+            const {document} = window;
+            for (const testElement of /** @type {NodeListOf<HTMLElement>} */ (document.querySelectorAll('test-case[data-test-type=scan]'))) {
+                test(`test-case-${testIndex++}`, () => {
+                // Get test parameters
```

Comment:
Check indentation

Reply:
@Scrub1492 Whenever you have a moment ^

(Also, I wonder why our linting didn't catch this?)

Reply:
Because it's technically legal, eslint seems to require alignment with either the previous or next line.

https://github.com/eslint/eslint/issues/8931